bcgov/biohubbc
# SIMSBIOHUB-403: Telemetry Table Upload

## Links to Jira Tickets

[SIMSBIOHUB-403](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-403)

## Description of Changes

- Added support for telemetry table to upload CSV files

## Testing Notes

- The linked ticket has an example file that can be uploaded
- The column validation will work similar to the observations (lat, latitude are both acceptable)
- If no deployment can be found from the uploaded data, an error will be thrown and the process will be stopped


Steps to test: 
1. Once the app is running, navigate to a survey
2. Create an animal in the `Marked and Known Animals` section
3. Back at the survey, open the telemetry table with the `Manage Telemetry` button
4. Add a deployment with the `Add` button in the left hand panel, select the animal you created earlier
5. Copy the device ID from the created deployment and remember the dates associated to the deployment
6. Download and open the `supplied_telemetry` on the ticket
7. Modify device id column in the csv file with the one you copied earlier
8. Ensure the dates in the file also fit within the date range on the deployment
9. Upload that CSV file
